tools/generate-llms-files.py

```
@@ -0,0 +1,336 @@
+
+                xml_lines.append(f'    <file name="{item_title}" url="{item_url}" description="{item_desc}">')
+
+                # Check if file is local and fetch its content
+                if base_dir and not item["url"].startswith(("http://", "https://", "mailto:")):
+                    file_path = os.path.join(base_dir, item["url"])
+                    if os.path.exists(file_path) and os.path.isfile(file_path):
+                        try:
+                            with open(file_path, "r", encoding="utf-8") as f:
+                                file_content = f.read()
+                            xml_lines.append(xml_escape(file_content))
+                        except Exception as e:
```

Ruff (0.16.1)

[warning] Do not catch blind exception: `Exception`

(BLE001)